repose-aggregator/functional-tests/spock-functional-test/src/test/groovy/features/filters/ratelimiting/RateLimitingConflictingIDsMultiFilterTest.groovy

```diff
@@ -0,0 +1,112 @@
+/*
+ * _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+ * Repose
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Copyright (C) 2010 - 2015 Rackspace US, Inc.
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+ */
+package features.filters.ratelimiting
+
+import framework.ReposeValveTest
+import groovy.json.JsonSlurper
+import org.rackspace.deproxy.Deproxy
+import org.rackspace.deproxy.MessageChain
+import spock.lang.Unroll
+
+class RateLimitingConflictingIDsMultiFilterTest extends ReposeValveTest {
+    final Map<String, String> userHeaderDefault = ["X-PP-User": "user"]
+    final Map<String, String> groupHeaderDefault = ["X-PP-Groups": "customer"]
+    final Map<String, String> acceptHeaderJson = ["Accept": "application/json"]
+
+    def setupSpec() {
+        deproxy = new Deproxy()
+        deproxy.addEndpoint(properties.targetPort)
+
+        def params = properties.getDefaultTemplateParams()
+        repose.configurationProvider.applyConfigs("common", params)
+        repose.configurationProvider.applyConfigs("features/filters/ratelimiting/idconflict", params)
+        repose.start()
+    }
+
+    def cleanupSpec() {
+        if (repose)
+            repose.stop()
+        if (deproxy)
+            deproxy.shutdown()
+    }
+
+    //2233
```

Comment:
If this is being called out as the fix for REP-2233, then it should actually say that.